### DISCUSSION TITLE
Get slot resources : ClientAccessForbiddenException - OCAPI

### DISCUSSION THREAD

**Discussion by gania241** [SalesforceCommerceCloud/sfcc-ci#547] [Team Posts]
Hi Folks,

I am trying to get slot details with slot data resources but looks I am getting ClientAccessForbiddenException in spite provided correct access OAuth token .

URL : https://xyz-alliance-prtnr-eu09-dw.demandware.net/s/-/dw/data/v19_5/sites/RefArch/slots/home-main-m/global?client_id=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
Provided by : OAuth Token

What could be the issue?

**Comment by pravngaur** [comment]
Please check the client id is correctly configured in BM ocapi settings.

**Comment by gania241** [comment]
This wasn't a client id issue..issue been fixed by mentioning individual resource for slots in OCAPI settings in BM.

**Comment by pravngaur** [comment]
Yes, all such exceptions related to access has to do with client id or the
allowed resources

**Comment by mpatelAshley** [comment]
I have the same issue. And Below setting i have in BM OCAPI settings for Data.

{
  "_v":"19.5",
  "clients":
  [ 
    {
      "client_id":"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
      "allowed_origins":["http://dev02-store.demandware.net","https://dev02-store.demandware.net"],
      "resources":
      [
        {
          "resource_id":"/customers",
          "methods":["post"],
          "read_attributes":"(**)",
          "write_attributes":"(**)"
        },
        {
          "resource_id":"/product_search",
          "methods":["post"],
          "read_attributes":"(**)",
          "write_attributes":"(**)"
        }
	   ]
	}
   ]
}
